jobs/build/sweep/Jenkinsfile

```diff
@@ -60,7 +65,20 @@ node {
             currentBuild.description += "* Searching for and attaching new bugs in MODIFIED state...\n"
             buildlib.elliott "--group=openshift-${version} find-bugs --mode sweep --use-default-advisory ${kind}"
         } catch (elliottErr) {
-            currentBuild.description = "Error sweeping:\n${elliottErr}"
+            currentBuild.description += "Error sweeping bugs:\n${elliottErr}"
+            throw elliottErr
+        }
+    }
+    stage("Sweep builds") {
+        if (!params.SWEEP_BUILDS) {
+            currentBuild.description += "* Not sweeping builds\n"
+            return
+        }
+        try {
+            currentBuild.description += "* Searching for and attaching new builds...\n"
+            buildlib.elliott "-g openshift-${version} find-builds -k ${kind} --use-default-advisory ${kind}"
```

Comment:
IIRC, current elliott find-builds command may have problems with attaching rhel8 builds.
There is a JIRA issue I created for the `elliott find-builds -b $build -a $advisory` situation: https://jira.coreos.com/browse/ART-1246

Reply:
So this will get fixed when that does :)

What I'm wondering here though is why only one `kind`. I envisioned both being swept.